CONTRIBUTORS

```diff
@@ -26,6 +26,7 @@ Johannes Maria Frank
 Chris Gorman
 Fernando Alvarruiz
 Tasos Papastylianou
+Jing-Chen Peng
```

Comment:
@mass2010chromium Is this how you'd like you name to appear in the credits?

(I know this is a very old pull request and I apologize for taking so long to get to it, but thank you for your contributions!)

Reply:
works for me, thank you!